app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Article</h1>
+
+<%= render 'form', article: @article %>
+
+<%= link_to 'Show', @article %> |
+<%= link_to 'Back', articles_path %>
```

Comment:
@sansuaza Please add the missing blank space at the end of the file

Reply:
okay